zetaclient/chains/evm/observer/inbound.go

```diff
@@ -34,6 +35,76 @@ import (
 	"github.com/zeta-chain/node/zetaclient/zetacore"
 )
 
+var (
+	// erc20AddressToForeignCoinAssetMap maps the chain id and foreign ERC20 address to the coin asset string
+	erc20AddressToForeignCoinAssetMap = map[int64]map[ethcommon.Address]string{
+		// Ethereum mainnet
+		chains.Ethereum.ChainId: {
+			// USDC.ETH
+			ethcommon.HexToAddress("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"): "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+			// PEPE.ETH
+			ethcommon.HexToAddress("0x6982508145454ce325ddbe47a25d4ec3d2311933"): "0x6982508145454ce325ddbe47a25d4ec3d2311933",
+			// SHIB.ETH
+			ethcommon.HexToAddress("0x95ad61b0a150d79219dcf64e1e6cc01f0b64c4ce"): "0x95ad61b0a150d79219dcf64e1e6cc01f0b64c4ce",
+			// USDT.ETH
+			ethcommon.HexToAddress("0xdac17f958d2ee523a2206206994597c13d831ec7"): "0xdac17f958d2ee523a2206206994597c13d831ec7",
+			// DAI.ETH
+			ethcommon.HexToAddress("0x6b175474e89094c44da98b954eedeac495271d0f"): "0x6b175474e89094c44da98b954eedeac495271d0f",
+			// ULTI.ETH
+			ethcommon.HexToAddress("0x0E7779e698052f8fe56C415C3818FCf89de9aC6D"): "0x0E7779e698052f8fe56C415C3818FCf89de9aC6D",
```

Comment:
```suggestion
```
Already in checksum format

Reply:
I made a full list, but reducing the list is also fine https://github.com/zeta-chain/node/pull/3273/commits/3bba7d378131eb07266a987acb18ba95b13f76fb